edxlearndot/signals.py

```diff
@@ -0,0 +1,66 @@
+"""
+Signal handling required for Learndot integration
+"""
+
+
+from __future__ import absolute_import, unicode_literals
+
+import logging
+
+from django.dispatch import receiver
+
+from openedx.core.djangoapps.signals.signals import COURSE_GRADE_NOW_PASSED
+
+from edxlearndot.learndot import EnrolmentStatus, LearndotAPIClient, LearndotAPIException
+from edxlearndot.models import CourseMapping
+
+
+log = logging.getLogger(__name__)
+
+log.info('edxlearndot.signals imported')
```

Comment:
nit: This shows up a few times on devstack startup and might be a bit verbose.

Reply:
Removed.